org.raceintospace.Raceintospace.yml

```diff
@@ -0,0 +1,89 @@
+# vim: sts=4:
+app-id: org.raceintospace.Raceintospace
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: raceintospace
+#rename-icon: raceintospace
+rename-desktop-file: raceintospace.desktop
+rename-appdata-file: raceintospace.appdata.xml
+modules:
+    - name: sdl
+      buildsystem: autotools
+      sources:
+        - type: archive
+          url: http://libsdl.org/release/SDL-1.2.15.tar.gz
+          sha256: d6d316a793e5e348155f0dd93b979798933fb98aa1edebcc108829d6474aad00
+        - type: patch
+          path: SDL-XData32.patch
+      cleanup:
+        - SDL*.3
+        - '/lib/*.a'
+        - '/include'
+    - name: jsoncpp
+      buildsystem: cmake
+      sources:
+        - type: git
+          url: https://github.com/open-source-parsers/jsoncpp.git
+          tag: 1.9.3
+      cleanup:
+        - '/lib/*.a'
+        - '/include'
+    - name: physfs
+      buildsystem: cmake
+      sources:
+        - type: archive
+          url: https://icculus.org/physfs/downloads/physfs-3.0.2.tar.bz2
+          sha256: 304df76206d633df5360e738b138c94e82ccf086e50ba84f456d3f8432f9f863
+    - name: protobuf
+      buildsystem: autotools
+      sources:
+        - type: git
+          url: https://github.com/protocolbuffers/protobuf.git
+          tag: v3.13.0
+      cleanup:
+        - '/lib/libproto*.a'
+        - '/include'
+    - name: protobuf-c
+      buildsystem: autotools
+      sources:
+        - type: git
+          url: https://github.com/protobuf-c/protobuf-c.git
+          tag: v1.3.3
+    - name: boost
+      # Borrowed boost build from https://github.com/flathub/org.kde.digikam
+      buildsystem: simple
+      build-commands:
+        - ./bootstrap.sh --prefix=/app
+        - ./b2 --with-headers -j $FLATPAK_BUILDER_N_JOBS
+        - ./b2 install
+      cleanup:
+        - /lib/libboost_*.a
+        - '/include'
+      sources:
+        - type: archive
+          url: https://dl.bintray.com/boostorg/release/1.74.0/source/boost_1_74_0.tar.bz2
+          sha256: 83bfc1507731a0906e387fc28b7ef5417d591429e51e788417fe9ff025e116b1
+    - name: raceintospace
+      buildsystem: cmake
+      sources:
+        - type: git
+          url: https://github.com/raceintospace/raceintospace.git
+      cleanup:
+        - '/include'
+        - '/lib/lib*.a'
+    # Icon should be in different place build system puts it, workaround
+    # .xpm icons are unsupported
+    - name: icon-rename
```

Comment:
Any reason not to use `post-install` in `raceintospace` module to run these commands?

Reply:
It might work. I could not find any documentation related to post-install however. I was just following https://docs.flatpak.org/en/latest/manifests.html when preparing this flatpak. Can you point me to some documentation how to use it?

Reply:
Okay, it just works. Thanks! But again, I found no information about it in documentation.